geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/index/AttributeIdxStrategy.scala

```diff
@@ -381,40 +409,21 @@ object AttributeIdxStrategy extends StrategyProvider {
   }
 
   def tryMergeAttrStrategy(toMerge: QueryFilter, mergeTo: QueryFilter): QueryFilter = {
-    // TODO: check disjoint range queries on an attribute
-    // e.g. 'height < 5 OR height > 6'
-    tryMergeDisjointAttrEquals(toMerge, mergeTo)
-  }
-
-  def tryMergeDisjointAttrEquals(toMerge: QueryFilter, mergeTo: QueryFilter): QueryFilter = {
-    // determine if toMerge.primary and mergeTo.primary are all Equals filters on the same attribute
     // TODO this will be incorrect for multi-valued properties where we have an AND in the primary filter
-    if (isPropertyIsEqualToFilter(toMerge) && isPropertyIsEqualToFilter(mergeTo) && isSameProperty(toMerge, mergeTo)) {
-      // if we have disjoint attribute queries with the same secondary filter, merge into a multi-range query
-      (toMerge.secondary, mergeTo.secondary) match {
-        case (Some(f1), Some(f2)) if f1.equals(f2) =>
-          mergeTo.copy(primary = mergeTo.primary ++ toMerge.primary, or = true)
+    val leftAttributes = toMerge.primary.map(FilterHelper.propertyNames(_, null))
```

Comment:
added - this method is used in the expand/reduce query processing that we try to avoid now - it's only used as a fallback if we can't figure out anything better.